README.md

```diff
@@ -57,10 +57,40 @@ If you are using Maven, add this to your pom.xml file
   <version>0.21.1-alpha</version>
 </dependency>
 ```
+Optionally, if you encounter dependency conflicts, add this to your pom.xml file to specify google-cloud-pom as ["bill of materials"](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies) in your project
```

Comment:
- End the line with a colon
- Add the article "a" before `["bill of materials"]` (same below too)